src/containers/statistics/Statistics.js

```diff
@@ -1,11 +1,77 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Statisitcs = () => {
-  return (
-    <div>
-      <h3>Aggregated statistics</h3>
-    </div>
-  );
+/* eslint-disable no-unused-vars */
+const allEmployees = (data) => {
+  const allEmployees = [];
+  data.forEach((comp) => {
+    if (comp.employees !== undefined) {
+      return allEmployees.push(comp.employees);
+    }
+  });
+  return allEmployees;
 };
 
-export default Statisitcs;
+const employeesTotal = (allEmployees) => {
+  allEmployees.reduce(function(accumulator, currentValue) {
+    return accumulator + currentValue;
+  });
+};
+
+const averageEmployees = (employeesSum, allemployees) => {
+  const compEmployeesCounter = allemployees.length;
+  return Math.round(employeesSum / compEmployeesCounter);
+};
+
```

Comment:
you used 3 different functions to calculate total and average no of employees, wouldn't it be possible to do it all within one function?

Reply:
good point, does not really make sense to iterate over an array of objects 3 times, could implement it with one loop that sets of data for example